web/src/message_viewport.js

```diff
@@ -389,6 +389,19 @@ export function recenter_view($message, {from_scroll = false, force_center = fal
     }
 }
 
+export function view_message_top($message) {
```

Comment:
Can we call this `scroll_to_show_message_top`? I think that name with the clear action verb would feel more readable.

Reply:
Sure, I like that name more.

Reply:
After some consideration with the previous review, I had decided to name it `maybe_scroll_to_show_message_top` instead as the function doesn't always scroll to align the viewport with the top of the message.

Reply:
That sounds great!